cli/cliui/agent_test.go

```diff
@@ -44,12 +44,14 @@ func TestAgent(t *testing.T) {
 			return err
 		},
 	}
-	cmd.SetOutput(ptty.Output())
-	cmd.SetIn(ptty.Input())
+
+	inv := cmd.Invoke()
```

Comment:
Could be `Request` or `Context`.

Reply:
I agree the name isn't great but I don't think Request or Context is better. Are you OK if we table this issue?

Reply:
Sure, that's fine. Renaming it is not worth delaying this PR considering the scope. 👍🏻

Other naming ideas: `Instance` (a configured/parameterized instance of the command), `Runner` (since it has `.Run` and runs the command).